resources/views/details-box.blade.php

```diff
@@ -13,28 +13,28 @@
     'tooltip' => null,
 ])
 
-<div class="flex items-center detail-box {{ $class }} @if ($reverse) flex-row-reverse @endif">
+<div class="flex detail-box {{ $class }} @if ($reverse) flex-row-reverse @endif">
     @if ($iconWrapper)
         {{ $iconWrapper }}
     @elseif ($icon || $iconRaw)
         @if ($noBorder)
-            <div class="flex-shrink-0 text-theme-secondary-900 dark:text-theme-secondary-600 {{ $iconWrapperClass }} @unless ($reverse) mr-5 @else ml-5 @endif">
+            <div class="{{ in_array('items-center', explode(' ', $class), true) ? '' : 'mt-0.5' }} flex-shrink-0 text-theme-secondary-900 dark:text-theme-secondary-600 {{ $iconWrapperClass }} @unless ($reverse) mr-5 @else ml-5 @endif">
```

Comment:
what does it need the `mt-0.5` for when it's not centered? shouldn't the parent pass that with the `class` property if it needs different handling? 🤔

Reply:
alternatively it would be better to have positions if we want to keep things consistent, so you can pass `centered` (default) or `top` and it handles styling based on that, instead of adding a margin if it does not find `items-center` (say we end up having a need for it being styled at the bottom, it would mean the `mt-0.5` will interfere as we might need a `mb-0.5` instead)

Reply:
I didn't think about the bottom position. The best solution is to add a position property as you proposed.